src/components/OrgPostCard/OrgPostCard.spec.tsx

```
@@ -1,980 +1,331 @@
 
-    const postModal = await screen.findByTestId('post-modal');
-    expect(postModal).toBeInTheDocument();
+global.URL.createObjectURL = vi.fn(() => 'mock-url');
+
+const sanitizeUrl = (url: string | null): string => {
+  if (!url) return '';
+
+  // For blob/data URIs - restrict to specific known safe patterns
+  if (url.startsWith('blob:')) {
+    // Only allow blob URLs from same origin
+    return url.startsWith('blob:' + window.location.origin) ? url : '';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)